com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ClientReleaser.java

```diff
@@ -42,7 +42,7 @@ public void releaseClient() {
         }
     }
 
-    protected String getCurrentUser(HistoryService historyService, String processInstanceId) {
+    public String getCurrentUser(HistoryService historyService, String processInstanceId) {

```

Comment:
This method does not belong to this class. The fact that it was `private` and we had to make it `public` when it doesn't fit the rest of the interface is an indication of that. Move it to somewhere else - it will probably fit best in a utility class.